.travis/.travis-ci.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+function prepare () {
+  sudo apk add m4 linux-headers
+  opam install dune ocamlfind core ounit qcheck react ppx_deriving yojson ounit ocp-indent
+  eval $(opam env)
+}
+
+function run_generator () {
+  sudo git clone https://github.com/exercism/problem-specifications.git ../problem-specifications
+  cd ../problem-specifications
+  sudo git checkout 2af3c9b0074f16c62366c5c533eaacd3ff27b583 
```

Comment:
My thinking was to avoid upates in `exercism/problem-specifications` break our build. 
Also this documents what git version the currently checked in test files relate to. 

The tradeoff here is the possibility to fall behind `exercism/problem-specifications` - I deemed it acceptable in exchange for higher CI stability. What do you think?

Reply:
The way [Haskell track solves this](https://github.com/exercism/haskell/blob/master/bin/ensure-readmes-are-updated.sh#L20) is to only perform the check for exercises that are touched by the PR. I have meant to push this script to `exercism/ocaml` once #309 is merged, but I'll wait until this PR is resolved, since we also address updates to the tests.

We could also run the script for all exercises, but only fail the CI if exercises that are touched by the PR. It seems that [Travis build status images](https://docs.travis-ci.com/user/status-images/) cannot reveal granular statuses, just a "![build: passing](https://travis-ci.org/exercism/ocaml.svg?branch=master)", so meh.

Eventually we can also build #284 into it and check version numbers from `exercism/problem-specifications` against a `(version ...)` in the `dune-project` file, but that is getting ahead.

Reply:
Also, with your work, the work in #284 is lowered considerably.

Reply:
Sorry, I don't quite gather if you think we point to an explicit git-ref or not?

Reply:
In the erlang track I have added the problem-specification repository as a submodule, this way I can rely on the fact that on a regular recursive clone everyone will see the same SHA.